linebreak.go

```diff
@@ -18,15 +18,15 @@ func Wrap(text string, width, maxwidth int) string {
 
 	minima := make([]int, count+1)
 	for i := 1; i < len(minima); i++ {
-		minima[i] = 1000000000000000000
+		minima[i] = 1 << 30
 	}
 	breaks := make([]int, count+1)
 
 	// closes over offsets, minima
 	cost := func(i, j int) int {
 		w := offsets[j] - offsets[i] + j - i - 1
 		if w > maxwidth {
-			return 10000000000 * (w - width)
+			return (1 << 30) * (w - width)
```

Comment:
On build where `int` is only 32-bits, this computation will overflow at runtime instead if `w - width > 4`.

A better fix for portability it to switch `int` to `int64` instead.